handlers.go

```diff
@@ -60,6 +61,37 @@ func AddBeer(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
 }
 
 // AddBeerReview adds a new review for a beer
-func AddBeerReview(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
-	// TODO
+func AddBeerReview(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	w.Header().Set("Content-Type", "application/json")
+
+	ID, err := strconv.Atoi(ps.ByName("id"))
+	if err != nil {
+		w.WriteHeader(http.StatusBadRequest)
+		json.NewEncoder(w).Encode(fmt.Sprintf("%s is not a valid Beer ID, it must be a number.", ps.ByName("id")))
+		return
+	}
+
+	for _, v := range Cellar {
+		if v.ID == ID {
+			decoder := json.NewDecoder(r.Body)
+
+			var newReview model.Review
+			err := decoder.Decode(&newReview)
+
+			newReview.BeerID = ID
```

Comment:
I'd probably move this line to the "successful" else block below (just above l.88), since we don't really care about setting this if there's an error from decoding.